Nagstamon/QUI/__init__.py

```diff
@@ -1620,9 +1622,11 @@ def hide_window_if_not_under_mouse(self):
         """
         Hide window if it is under mouse
         """
-        mouse_x = QCursor.pos().x()
-        mouse_y = QCursor.pos().y()
-        if statuswindow.geometry().contains(mouse_x, mouse_y):
+        mouse_pos = QCursor.pos()
+
+        # Check mouse cursor over window and an opened context menu or dropdown list
+        if statuswindow.geometry().contains(mouse_pos.x(), mouse_pos.y()) \
+                or not qApp.activePopupWidget() is None:
```

Comment:
Where does qApp come from? Shouldn't it be APP as defined in line 30?

Reply:
It is `QtWidget.qApp`. You have already imported everything from `QtWidget`. I assume `APP` is the same here.
https://doc.qt.io/qt-5/qapplication.html#qApp

Reply:
Yes, APP is the current instance there.